storage/google/cloud/storage/bucket.py

```diff
@@ -398,7 +398,7 @@ class Bucket(_PropertyMixin):
     https://cloud.google.com/storage/docs/storage-classes
     """
 
-    def __init__(self, client, name=None, user_project=None):
+    def __init__(self, client=None, name=None, user_project=None):
```

Comment:
Let's make this update as a separate PR, as I think we'll want to show better error messages in the existing Bucket methods when `client` isn't supplied (for example, say what method to call instead on Client).

Reply:
Took this file out of review - please disregard, thanks!